OPAL/br/src/main/scala/org/opalj/br/fpcf/analyses/L0CompileTimeConstancyAnalysis.scala

```diff
@@ -48,11 +51,12 @@ class L0CompileTimeConstancyAnalysis private[analyses] ( final val project: Some
             return Result(field, CompileTimeConstantField);
 
         var dependee: EOptionP[Entity, Property] = {
-            propertyStore(field, FieldMutability.key) match {
-                case FinalP(LazyInitializedField) => return Result(field, CompileTimeVaryingField);
-                case FinalP(_: FinalField)        => return Result(field, CompileTimeConstantField);
-                case FinalP(_: NonFinalField)     => return Result(field, CompileTimeVaryingField);
-                case ep                           => ep
+            propertyStore(field, FieldAssignability.key) match {
+                case LBP(NonAssignable) => return Result(field, CompileTimeConstantField);
+                case UBP(EffectivelyNonAssignable |
```

Comment:
Why is effective non-assignability here and not grouped with full non-assignability?